src/Misc/externals.sh

```diff
@@ -127,6 +127,10 @@ function acquireExternalTool() {
     fi
 }
 
+# Download the external tools for all platforms
+acquireExternalTool "$CONTAINER_URL/NuGet/3.5.1-beta1-1621/NuGet.zip" nuget
```

Comment:
If the machine has mono installed and configured, yes, at least against nuget.org.